apps/antalmanac/src/lib/helpers.ts

```diff
@@ -42,7 +42,7 @@ export async function clickToCopy(event: React.MouseEvent<HTMLElement, MouseEven
 }
 
 export function isDarkMode() {
```

Comment:
This shouldn't exist anymore, and it's consumers should refactor to use the store directly. Up to you to include it in this PR.

Reply:
I think there's like 15 something files which call isDarkMode(), so I'd tentatively prefer to punt that down the line in a separate PR; I'll see how I feel about it once I address the other comments

Reply:
Yeah; I'll move this to another PR